packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -575,17 +582,10 @@ export class Dropdown {
      * @returns {JSX.Element}
      */
     renderButtonLabel(): JSX.Element {
+        const label = this.label ?? this.selected?.[0]?.text ?? this.selected?.[0]?.innerHTML ?? '';
         this.selected?.[0]?.text ?? this.selected?.[0]?.innerHTML ?? '';
 
-        if (this.multiple) {
-            return (<div part="button-label" class="button__label">{this.label}</div>);
-        } else if (this.selected?.[0]?.text) {
-            return (<div part="button-label" class="button__label">{this.selected[0].text}</div>);
-        } else if (this.selected?.[0]?.innerHTML) {
-            return (<div part="button-label" class="button__label button__label--from-inner-html" innerHTML={this.selected[0].innerHTML}></div>);
```

Comment:
Should we somehow preserve the styling given from the `--from-inner-html` modifier?